src/Aguacongas.AspNetCore.Authentication.TestBase/DynamicManagerTestBase.cs

```diff
@@ -34,7 +34,7 @@ namespace Aguacongas.AspNetCore.Authentication.TestBase
     /// </summary>
     /// <typeparam name="TSchemeDefinition">The type of the scheme definition.</typeparam>
     public abstract class DynamicManagerTestBase<TSchemeDefinition>
```

Comment:
The class should be renamed `AuthenticationSchemeProviderWrapperTestBase` IMO as `DynamicManager` doesn't exist any more